spec/marketing_only_app/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
+  "app": {
+    "name": {
+      "title": "Name for App: Marketing Only App",
+      "value": "Marketing Only App"
+    },
+    "description": {
+      "title": "Description for App: Marketing Only App",
+      "value": "Bring your Zendesk into your other service"
+    },
+    "long_description": {
+      "title": "Long Description for App: Marketing Only App",
+      "value": "Zendesk for Marketing Only App allows you to integrate your Zendesk Support helpdesk with your other service."
```

Comment:
The company rule is not to use 'help desk' when referring to Zendesk products. Zendesk makes customer service software, not help desk software. Suggested edit: 

> ...allows you to integrate your Zendesk with your other service.

It'll probably need to be changed to "allows you to integrate Zendesk Support with your other service" with GoLion in October.